src/py42/_internal/clients/detection_list_user.py

```diff
@@ -14,13 +15,29 @@ class DetectionListUserClient(BaseClient):
     _uri_prefix = u"/svc/api/{0}".format(_api_version)
     _resource = u"/user"
 
-    def __init__(self, session, user_context):
+    def __init__(self, session, user_context, user_client):
         super(DetectionListUserClient, self).__init__(session)
         self._user_context = user_context
+        self._user_client = user_client
 
     def _make_uri(self, action):
         return u"{0}{1}{2}".format(self._uri_prefix, self._resource, action)
 
+    def create_if_not_exists(self, user_id):
+        """Find whether detection list profile exists for given uid.
+            If not, if its a valid uid of code42 user then create detection list profile.
+
+            Args: Uid of user.
```

Comment:
The format of this docstring isn't quite right, compare it with others and adjust

Reply:
Changed.